api/src/Streams.cpp

```diff
@@ -285,7 +318,7 @@ void Streams::WriteStreamsJSON(
 }
 
 std::shared_ptr<Stream> Streams::Emplace(const Channel &channel) {
-  auto stream = std::make_shared<Stream>(db_, channel);
+  auto stream = std::make_shared<Stream>(db_, channel, channel);
```

Comment:
second channel here is the chat channel?

Reply:
oh yeah, this should have been copied from the user here. nice catch 👍